packages/cf.js/API_REFERENCE.md

```diff
@@ -11,6 +11,8 @@
             - [Node method](#method-getappinstances)
         - `async install(appInstanceId: AppInstanceID): Promise<AppInstance>`
             - [Node method](#method-install)
+        - `async installVirtual(appInstanceId: AppInstanceID, intermediaries: Address[]): Promise<AppInstance>`
```

Comment:
Would `intermediaries` be specified again? Could they not just be looked up via the `appInstanceId`?

Cuz I'm imagining the proposal happens through a specified intermediary, so doing the actual install via a possibly different intermediary would deviate from the proposal.